release.md

```diff
@@ -0,0 +1,173 @@
+   called `bottles`.
+
+ * **Windows:** currently *there is no binary distribution for Windows* that covers
+   all Ignition libraries although there is an ongoing effort for getting
+   binary packages inside
+   [conda-forge](https://github.com/conda-forge/staged-recipes/issues/13551)
+
+### Infrastructure
+
+The Ignition team uses different pieces of infrastructure to execute the
+releasing process:
+
+ * **build.osrfoundation.org**: main Jenkins server to orchestrate the whole
+   releasing process. It receives requests from Ignition developers to start the
+   release process, schedule nightly builds, build the binary packages,
+   upload releasing artifacts to repositories, etc.
+
+ * **packages.osrfoundation.org**: Debian/Ubuntu .deb repositories. The URLs
+   are ready to be used by the APT packages system.
+
+ * **osrf-distributions S3**: [cloud storage](http://gazebosim.org/distributions/bottles-simulation/)
+   space to save the different artifacts produced during the release process. The
+   S3 space is no more than archive of artifacts that need to be manually
+   downloaded (i.e: to get an old version of a .deb package) or handled by other
+   packaging systems (i.e: brew download the bottles from S3, the buildfarm uses
+   the tarballs stored in S3 when building .deb packages)
+
+### Type of releases
+
+The Ignition development uses three different types of releases depending on the
+stability of the software:
+
+ * **stable**: stable releases are designed to be generally consumed by users
+   that do not expect to solve severe issues or regressions.
+   * Linux stable binaries:
+     http://packages.osrfoundation.org/gazebo/{ubuntu/debian}-stable
+   * macOS stable binaries:
+     http://gazebosim.org/distributions/ (in their respective directories)
+
+ * **prerelease**: prereleases are designed to be tested as candidates to
+   stable.
+   * Linux prerelease binaries:
+     http://packages.osrfoundation.org/gazebo/{ubuntu/debian}-prerelease
+   * macOS prerelease binaries (they are stored in the same place as stable releases but under bottles-simulation):
+     http://gazebosim.org/distributions/bottles-simulation/
+
+ * **nightly**: automatically generated releases used in continuous integration
+   systems reflecting the latest changes in the repositories.
+   * Linux nightly binaries:
+     http://packages.osrfoundation.org/gazebo/{ubuntu/debian}-nightly
+   * macOS nightly binaries:
+     *Unsupported* (see [the homebrew-simulation issue](https://github.com/osrf/homebrew-simulation/issues/1314) 
+     for more information)
+
+### Metadata for Releasing
+
+All package managers need to define different data fields in order to provide
+the features needed to install and remove packages. In the case of Ignition
+Libraries supported packages:
+
+ * **deb packages**: debian packaging metadata for Debian/Ubuntu packages is
+     stored in a custom way inside the `release` repositories. At the moment of
```

Comment:
```suggestion
     stored in a custom way inside the `$LIBRARY-release` repositories. At the moment of
```
It might be a little clearer to indicate that there's a release repository for every library.